### DISCUSSION TITLE
Resizing the vue echart without affecting the DOM

### DISCUSSION THREAD

**Discussion by vito-gmi** [ecomfe/vue-echarts#720] [Q&A]
Hi Community,

Does anyone know if there is a way to resize the vue-echart without affecting the DOM, but still have my v-chart component auto resizable?

The idea is that I am exporting the chart to word document. However, when I am using a monitor, the size of the chart is much bigger and I have found only way to handle this size with using the `resize()` method. However, when I resize my chart to a fixed `width` and `height` it also affects my DOM, in my case, it cuts part of the data.

Before:
![image](https://github.com/ecomfe/vue-echarts/assets/25567145/8e89caa2-59d7-466b-945f-26a9381aab24)

After:
![image](https://github.com/ecomfe/vue-echarts/assets/25567145/16dddaf6-fe85-4552-a9cf-9f03f8e984e7)

**Comment by Justineo** [comment]
Not sure what you mean by "without affecting the DOM" specifically. If you are using the `autoresize` prop the graph probably shouldn't be cut off like this.

**Comment by vito-gmi** [comment]
@Justineo well, the thing is that when I export the chart to word file, it expands the chart too much when exported from bigger screens. The chart acquire egg shape, not circled like it is seen on the web page. When I shrink the screen size, then it is exported perfectly. Therefore, I decided to resize the chart before exporting it. However, this way my chart got corrupted, as it can be seen above. I want to keep my chart resizable based on the webpage screen, but still not affected from the resizing before export.

I have found workaround, but do not know if that is the best way handling my problem. I just added second chart which is `display: none;` and I am resizing and copying this one, instead of the original that is visible to the user.
